apps/console/src/extensions/configs/identity-provider.ts

```diff
@@ -44,6 +45,28 @@ export const identityProviderConfig: IdentityProviderConfig = {
                 IdentityProviderManagementConstants.X509_AUTHENTICATOR,
                 IdentityProviderManagementConstants.SESSION_EXECUTOR_AUTHENTICATOR
             ].includes(name);
+        },
+        isProvisioningAttributesEnabled(authenticatorId: string): boolean {
+            const excludedAuthenticators = new Set([]);
+            /**
+             * If the {@param authenticatorId} is not in the excluded set we
+             * can say the provisioning attributes is enabled for authenticator.
+             */
+            return !excludedAuthenticators.has(authenticatorId);
```

Comment:
Let's return `false` here and add this as a doc comment.

Reply:
This has been addressed in commit c8571bd.